### PR TITLE
prettify demo

### PR DESCRIPTION
**Pull request by notwaldorf**
(This is a super old branch I had opened before the holidays)
Added `demo-snippet`. Had to temporarily comment out the `dom-bind` demo, since it's busted on IE (because of nested templates in `webcomponents.js`), but I'll add it back as soon as that's fixed.

Looks like this (and more):
screen shot 2016-01-14 at 12 31 06 pm

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -19,191 +19,202 @@
...
-      'rajasaurus',
-      'stegosaurus',
-      'triceratops',
-      'utahraptor',
-      'vulcanodon',
-      'wannanosaurus',
-      'xenoceratops',
-      'yandusaurus',
-      'zephyrosaurus'
+      "allosaurus",
```
Comment (cdata) [inline demo/index.html]: Why did you change all of these quotes? I thought we prefer `'` over `"` in JS..

**Comment by cdata** [comment]
Small question, otherwise LGTM :+1:

File: demo/index.html (hunk 1 above)
Comment (notwaldorf) [inline demo/index.html]: hoy crap, good catch. i think i did a bad find-and-replace (wasn't intentional)

**Comment by addyosmani** [comment]
LGTM2 once the quotes have been fixed :+1:

**Comment by notwaldorf** [comment]
I fixed the delinquent quotes!
